src/include/daos/tse.h

```diff
@@ -405,4 +405,10 @@ tse_task_depend_list(tse_task_t *task, d_list_t *head);
 int
 tse_task_list_traverse(d_list_t *head, tse_task_cb_t cb, void *arg);
 
+/**
+ * Set the task don't propagate err-code from dependent tasks.
+ */
+void
+tse_set_dont_propagate(tse_task_t *task);
```

Comment:
would suggest to change the name of this function similarly to Jeff's suggestion. tse_avoid_propagate() or something.
since this is a public function, probably better to refresh the patch before landing to avoid an API change in the future.

Reply:
ok, refreshed.